app/views/notifications/_nav_menu.html.erb

```diff
@@ -1,72 +1,42 @@
-<nav class="hidden m:block pt-4" aria-label="Notifications">
-  <div>
-    <% if @organizations.present? %>
-      <h3 class="crayons-subtitle-3 mb-4 pl-2"><%= @user.name %></h3>
-    <% end %>
-    <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].blank? %>" href="/notifications">
-      All
-    </a>
-    <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].blank? %>" href="/notifications/comments">
-      Comments
-    </a>
-    <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("posts").zero? && params[:org_id].blank? %>" href="/notifications/posts">
-      Posts
-    </a>
-  </div>
+<nav class="px-3 s:px-0" aria-label="Notifications">
   <% if @organizations.present? %>
-    <% @organizations.each do |org| %>
-      <div class="pt-6">
-        <h3 class="crayons-subtitle-3 mb-4 pl-2"><%= org.name %></h3>
-        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("org").zero? && params[:org_id].to_i == org.id %>" href="/notifications/org/<%= org.id %>">
-          All
-        </a>
-        <a class="crayons-link crayons-link--block <%= "crayons-link--current" if params[:filter].to_s.casecmp("comments").zero? && params[:org_id].to_i == org.id %>" href="/notifications/comments/<%= org.id %>">
-          Comments
-        </a>
-      </div>
-    <% end %>
-  <% end %>
-</nav>
-
-<nav class="block m:hidden px-4" aria-label="Notifications">
-  <% if @organizations.present? %>
-    <select id="notifications-filter__select" class="crayons-select mb-2">
+    <select id="notifications-filter__select" class="crayons-select mb-4" aria-label="Filter by account...">
       <option value="/notifications"><%= @user.name %></option>
       <% @organizations.each do |org| %>
         <option value="/notifications/org/<%= org.id %>" <%= "selected" if params[:org_id].to_i == org.id %>><%= org.name %></option>
       <% end %>
     </select>
```

Comment:
It would be great if we could avoid using a `select` for navigation - it stops the elements from being links, which is key for accessibility given it triggers a page change. Have we considered adopting a similar pattern to the new home feed/search tabs (i.e. the two rows of links, orgs on the top if relevant, then All/Comments/etc)?

Reply:
Dropping selects is my ultimate goal generally but in this case I think it makes some sense... I'm not trying to convince you to using `<select>` element because I know its suboptimal. It could be even a JS `crayons-dropdown` - the point is to have this kind of interface. I can't think of a better UI/UX pattern for this. But I'll give it more thought...

Having two rows of tabs in this context is a bit weird because both rows are very related (here we have two rows of filters, as in search results both rows have different purpose: sorting & filtering).

Reply:
A dropdown with a nav inside would float my boat ⛵ 

This was the route we'd headed down with the [mobile nav drawer](https://storybook.forem.com/?path=/story/app-components-mobiledrawernavigation--default), with the checkmark indicating the current page (and a separate heading to show what page we were currently on when the dropdown wasn't open):

<img width="498" alt="screenshot of a dialog with some links, one has a checkmark and is coloured purple" src="https://user-images.githubusercontent.com/20773163/133803094-16ec6666-a3e2-4947-a0df-78c2efeacd3d.png">